### DISCUSSION TITLE
Display support for ESP32-C3, ESP32-S2, ESP32-S3

### DISCUSSION THREAD

**Discussion by lboue** [arendst/Tasmota#18552] [Support request]
Hello,

Could we add display support for ESP32-C3, ESP32-S2, ESP32-S3 to web page [release binaries for Tasmota firmware on ESP32](https://ota.tasmota.com/tasmota32/release/)?

I would connect SSD1306 OLED Display but I can.
We could have :
tasmota32c3cdc-display.bin
tasmota32c3cdc-display.bin
tasmota32s3cdc-display.bin

Should I only add `#define USE_DISPLAY_SSD1306` to user_config_override.h file to enable display and build it myself ?

Regards

**Comment by sfromis** [comment]
Better not cross-post your questions, already handled on Discord

**Comment by lboue** [comment]
Yes I know. I managed to make it work but compiling manually.
Would it be possible soon to make these files available as standard?

**Reply by andrico21** [reply]
+1 actually, is there a standard binary for C3U with display?
